helm-chart/csi-driver/values.yaml

```diff
@@ -1,13 +1,19 @@
 # Secrets:
-# apiToken [Required] - Must be a Linode APIv4 Personal Access Token with all permissions. (https://cloud.linode.com/profile/tokens) 
+# apiToken [Required if secretRef is not set] - Must be a Linode APIv4 Personal Access Token with all permissions. (https://cloud.linode.com/profile/tokens)
 apiToken: ""
 
-# region [Required] - Must be a Linode region. (https://api.linode.com/v4/regions)
+# region [Required if secretRef is not set] - Must be a Linode region. (https://api.linode.com/v4/regions)
 region: ""
 
 # Default namespace is "kube-system" but it can be set to another namespace
 namespace: kube-system
 
+# Set these values if your APIToken and region are already present in a k8s secret.
+# secretRef:
+#   name: "linode"
+#   apiTokenRef: "apiToken"
+#   regionRef: "region"
+
```

Comment:
Question: Are we going to set these values when we install the helm chart? If yes, why is this commented?

Reply:
Yes, these values can be passed in when installing. This is commented to keep backwards compatibility. Existing users of the chart can continue to pass in the token and region, new users can choose to use the secretRef. Didn't want to break users